app/app.component.ts

```diff
@@ -21,26 +16,23 @@ declare var android;
 */
 @Component({
   selector: "my-app",
-  providers : [UserService],
-  templateUrl: "pages/login/login.html",
-  changeDetection: ChangeDetectionStrategy.OnPush
+  templateUrl: "./pages/login/login.html"
 })
-export class AppComponent extends observableModule.Observable implements OnInit {
```

Comment:
You don't need to use and should not use {N}'s `Observable`. Just use rxjs observables. Don't ever extend and angular component to inherit from {N}'s observable.